<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWindow</class>
 <widget class="QMainWindow" name="AddStudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Student</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/assets/icons/logo.png</normaloff>:/icons/assets/icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">* {
	background-color: #e3f2fd;
}
#std_image {
	background-color: #fff;
}

#header_frame, #label {
	background-color: #90caf9;
}

#bottom_widget {
	background-color: #b3e5fc;
}

QPushButton {
	background-color: #42a5f5;
	border-radius: 5px;
	color: #fff;
}

#btn_cancel {
	background-color: #cfd8dc;
	border-radius: 5px;
	color: #fff;
}

QLineEdit {
	background-color: #fff;
	border-radius: 5px;
	border: 1px solid #039be5;
	padding: 5px 5px;
}

QComboBox {
	background-color: #fff;
	border-radius: 5px;
	border: 1px solid #039be5;
	padding: 5px 5px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="header_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add Student</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="std_image">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Addmission Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Reg / Adm No.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Student Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Father Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Date of Birth</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Contact #</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Gender</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Class</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Section</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Last School Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Special Case</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="txt_admission_date">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_admission_no">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_student_name">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_student_father_name">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_student_dob">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_student_address">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_student_contact">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_student_gender">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="select_class">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Select</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="select_class_section">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_last_school">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_special_case">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bottom_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/assets/icons/save.png</normaloff>:/icons/assets/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/assets/icons/close.png</normaloff>:/icons/assets/icons/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
